crates/orchestrator/src/jobs/state_update_job/mod.rs

```diff
@@ -222,3 +243,45 @@ impl StateUpdateJob {
         job.metadata.insert(new_attempt_metadata_key, tx_hashes.join(","));
     }
 }
+
+#[derive(Error, Debug)]
+pub enum StateUpdateError {
```

Comment:
let's move all errors to top of file

Reply:
unaware of this convention, Moved to top.